scripts/lib/api/processHtml.test.ts

```diff
@@ -0,0 +1,300 @@
+                <path fill-rule="evenodd" d="M18.78 15.28a.75.75 0 000-1.06l-6.25-6.25a.75.75 0 00-1.06 0l-6.25 6.25a.75.75 0 101.06 1.06L12 9.56l5.72 5.72a.75.75 0 001.06 0z"></path>
+            </svg>
+        </div>
+    </h3>
+    <div class="sd-summary-content sd-card-body docutils">
+        <div>
+                <p class="sd-card-text">You need to initialize your account before you can start using the Qiskit Runtime service...</p>
+                <p class="sd-card-text">Qiskit Runtime is available...</p>
+            </div>
+    </div>
+    </details>
+  `);
+});
+
+test("renameAllH1s()", () => {
+  const doc = Doc.load(`<h1>Release Notes!!!</h1><h2>0.45.0</h2>`);
+  renameAllH1s(doc.$, "New Title");
+  doc.expectHtml(`<h1>New Title</h1><h2>0.45.0</h2>`);
+});
+
+test("removeHtmlExtensionsInRelativeLinks()", () => {
+  const doc = Doc.load(
+    `<a href="https://ibm.com"></a><a href="../foo.html"></a><a href="../img.svg"></a>`,
+  );
+  removeHtmlExtensionsInRelativeLinks(doc.$, doc.$main);
+  doc.expectHtml(
+    `<a href="https://ibm.com"></a><a href="../foo"></a><a href="../img.svg"></a>`,
+  );
+});
+
+test("removeDownloadSourceCode()", () => {
+  const doc = Doc.load(
+    `<p>(<a class="reference download internal" download="" href="../_downloads/878087bb6064f5d5f4c00d6f8a12319d/converters-1.py"><code class="xref download docutils literal notranslate"><span class="pre">Source</span> <span class="pre">code</span></code></a>)</p>
+      <figure class="align-default">
+      <img alt="../_images/converters-1.png" class="plot-directive" src="../_images/converters-1.png" />
+      </figure>`,
+  );
+  removeDownloadSourceCode(doc.$main);
+  doc.expectHtml(
+    `<figure class="align-default">
+      <img alt="../_images/converters-1.png" class="plot-directive" src="../_images/converters-1.png">
+      </figure>`,
+  );
+});
+
+test("removePermalinks()", () => {
+  const doc = Doc.load(`<a title="okay">Link</a>
+    <a title="Permalink to this headline">Link</a>
+    <a title="Permalink to this heading">Link</a>
+    <a title="Permalink to this definition">Link</a>
+    <a title="Link to this heading">Link</a>
+    <a title="Link to this definition">Link</a>`);
+  removePermalinks(doc.$main);
+  doc.expectHtml(`<a title="okay">Link</a>`);
+});
+
+test("removeColonSpans()", () => {
+  const doc = Doc.load(
+    `<dt class="field-odd">Parameters<span class="colon">:</span></dt>`,
+  );
+  removeColonSpans(doc.$main);
+  doc.expectHtml(`<dt class="field-odd">Parameters</dt>`);
+});
+
+test("addLanguageClassToCodeBlocks()", () => {
+  const doc1 = Doc.load(`<p><strong>Circuit symbol:</strong></p>
+    <div class="highlight-default notranslate"><div class="highlight"><pre><span></span>     ┌──────────┐
+    q_0: ┤ U(ϴ,φ,λ) ├
+        └──────────┘
+    </pre></div>
+    </div>`);
+  // TODO(EA): This behavior is bad. We shouldn't be setting the language as Python because it's not always correct, such
```

Comment:
I'm not fixing this here because it will result in changes when we regenerate all the API docs.